setup.py

```diff
@@ -16,26 +16,30 @@
 )
 
 install_requires = [
-    "setuptools",
-    "plone.api >= 1.8.4",
     "hurry.filesize",
     "ijson",
+    "setuptools",
     "six",
 ]
 
 if sys.version_info[0] < 3:
+    install_requires.append("attrs < 22")
     install_requires.append("beautifulsoup4 < 4.10")
-    install_requires.append("plone.restapi < 8.0.0")
-    # There is a py2-imcompatibility in plone.rest 3.0.1
-    install_requires.append("plone.rest < 3.0.1")
     # plone.restapi depends on plone.schema, which depends on jsonschema,
     # which has a Py3-only release since September 2021.
     install_requires.append("jsonschema < 4")
-    install_requires.append("pyrsistent < 0.16.0")
     install_requires.append("pathlib2")
+    install_requires.append("plone.api >= 1.8.4, < 2")
+    # There is a py2-imcompatibility in plone.rest 3.0.1
+    install_requires.append("plone.rest < 3.0.1")
+    install_requires.append("plone.restapi < 8")
+    install_requires.append("plone.schema < 2")
+    install_requires.append("PyJWT < 2")
+    install_requires.append("pyrsistent < 0.16.0")
```

Comment:
It looks to me like you're adding pins for Plone 4, but doing it based on the presence of Python 2. It is also possible to use Python 2 with Plone 5.1 and 5.2

Reply:
I‌ tested only in Plone 4.
But above these thresholds there is only Python 3 support - also for plone.schema, plone.restapi and plone.api.
I think this should be safe.

Reply:
@thet I think you are right.